pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: carp_study_app
 description: The generic CARP study app.
 publish_to: 'none'
-version: 1.2.0
+version: 1.4.0
```

Comment:
If you change the version, please also change the changelog and the iOS version in XCode. I've updated the version in XCode.

Reply:
Should be done now. Also updated the README a bit.